config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
@mcastelino Sure. But, distros picking them up and getting them out (they may not backport either, and it may depend on a qemu release cycle as well as a distro update cycle) could take, literally, >1yr. So, what we are probably discussing here is what do we do in the meantime. We can't really wait that long before we wish to migrate to q35...